llvm/test/CodeGen/X86/GlobalISel/calllowering-nocrashret.ll

```diff
@@ -11,3 +11,14 @@ define <4 x i1> @foo() {
 entry:
   ret <4 x i1> undef ;
 }
+
+define <4 x i1> @foo_poison() {
```

Comment:
Better to move this to a more obviously related test file, either an irtranslator-poison.ll or test/CodeGen/GlobalISel/arm64-irtranslator.ll is used as the catch-all